elsafluss/rancid-tomatillos
# Cleanup

#### What does  this PR do?
Removes extra spaces, comments, checks indentation, etc. Also changed loading screen background to $dark instead of $highlight so it isn't as shocking.
#### Where should the reviewer start?
I checked all files to make sure they looked ok.
#### How should this be manually tested?
Pull it down and make sure the app still works locally, then we'll want to redeploy.
#### Any background context you want to provide?
